adapters/smaato/smaato.go

```diff
@@ -506,9 +521,33 @@ func setImpForAdBreak(imps []openrtb2.Imp) error {
 		imps[i].Video = &videoCopy
 	}
 
+	imps[0].Ext = impExt
+
```

Comment:
@ioan-alexandru-stef-smaato  any specific reason to only set `impExt` to `imps[0]` and not on other indexes

Reply:
@onkarvhanumante for AdPods, no matter how many imps we get, we only use one `Skadn` value. So it is usually the easiest to set it on only one `imp`. There's also an advantage I see which would be smaller payload.

Reply:
Should create documentation PR to include above `skadn` related behaviour

Reply:
Sure. https://github.com/prebid/prebid.github.io/pull/4592